sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+
+### Auxiliary Features
+Auxiliary features of the sBTC Bootstrap protocol are described below. 
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this [issue](https://github.com/stacks-network/stacks-core/issues/4235), sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
+- **Signer Key Rotation:** Mechanisms are provided for the scenario where a signer wants to rotate their key. For this to happen, signers must coordinate offline and vote on-chain on the new signer set (aka set of keys). Once the new signer set is determined, the signers conduct a wallet handoff and re-execute DKG.
+
+## sBTC Bootstrap Signers
+
+### Responsibilities
+The sBTC bootstrap signers are responsible for accepting or rejecting all sBTC operations submitted, and for a transaction to be fulfilled at least 70% of the signers need to approve the fulfilling of the transaction; this means that the liveness and reliability of the signers is crucial to the success of the protocol.
+
+While up to 30% of the signers can be down without any user impact to the functioning of the protocol, it becomes more critical for the rest of the signers to approve sBTC operations because operations necessarily still need to meet 70% of the original signing power. If more than 30% of signers become unavailable, no sBTC operations will be approved because it will be impossible to get the required 70% approval when less than 70% are online. 
+
+An operation that isn’t approved will become spendable by the user without bridging to the other blockchain after a period of time without Signer interaction.
+
+### Eligibility Criteria
+For the sBTC Bootstrap release Signers will run the sBTC binary in addition to the core Stacks signer software and must meet the following criteria in order to strongly ensure reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria has been used to identify the sBTC Bootstrap Signers:  
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet. 
+- **Technical Performance and Uptime:** Consistency in operational status and network stability. 
+- **Communication & Availability:** Running an sBTC Signer is a pivotal role in the Stacks ecosystem, which requires a high degree of availability and communication with Stacks core developers. Signers should generally be able to respond to updates within 24 hours. 
+- **Ecosystem Alignment:** Commitment to the growth of the Stacks ecosystem with contributions to support the network. Examples include (but are not limited to): publishing independent research or contributing to a Stacks working group.
```

Comment:
I have the same questions as before -- how do you measure these in an unambiguous, falsifiable manner?

I think this is of paramount importance for building trust in these signers.  How do other users determine that the selected signers actually fit these criteria?  I don't think they should just take our word for it, because for all they know, we could be playing favorites or running sockpuppets.  Let me put it this way -- how does a 3rd party (not us, not the voters) determine that a particular signer has meet these criteria? 

The SC will also need to know these criteria, because they'll need to evaluate signers with them in order to determine whether or not the SIP's activation terms have been met.